recipes/database.rb

```diff
@@ -1,15 +1,17 @@
 
```

Comment:
This file has a large chunk of commented out code. Remove if needed. Github won't let me add notes to comments.